foolbox/models/base.py

```diff
@@ -64,9 +64,11 @@ def _preprocess(self, inputs: ep.TensorType) -> ep.TensorType:
         min_, max_ = self._model.bounds
         return x * (max_ - min_) + min_
 
-    @property
-    def data_format(self) -> Any:
-        return getattr(self._model, "data_format", None)
+    def __getattr__(self, item: str) -> Any:
+        if item == "data_format":
+            return getattr(self._model, "data_format")
```

Comment:
I think you can just use `self._model.data_format`

Reply:
Then mypy detects an error as `data_format` is not defined in `Model`. I thought this would be a cleaner solution than to tell mypy to ignore this line.